connecting-to-testnet/building-binary.md

```diff
@@ -0,0 +1,64 @@
+---
+order: 1
+---
+
+# Building and Starting a Full Node
+
+## Requirements
+
+* [Go](https://golang.org/) (version 1.15 or higher)
+* [make](https://en.wikipedia.org/wiki/Make_(software))
+* [jq](https://github.com/stedolan/jq)
+* [git](https://git-scm.com/)
+* [curl](https://github.com/curl/curl)
+
+
+## Building the Binary
+
+Clone the code repository:
+
+```
+git clone --branch v1.0.1 https://github.com/b-harvest/gravity-dex
```

Comment:
Why gravity dex here instead of gaia? Should we consider adding this to the title?

Reply:
This is the source code of the testnet we're running. It's essentially Gaia with the liquidity module.

Reply:
Please add this sentence to the article so the user understands this as well 🙏

Reply:
Done. Feel free to edit the text directly. Also, @barriebyron if you can go over these docs, it would be super helpful! (not urgent, though)

Reply:
Should we clone this into a `testnet` directory. I find the directory name `gravity-dex` confusing for a `gaia` or `cosmos-sdk` testnet